azure/packages/azure-client/package.json

```diff
@@ -78,7 +78,7 @@
 		"@arethetypeswrong/cli": "^0.13.3",
 		"@fluid-tools/build-cli": "0.29.0-222379",
 		"@fluidframework/aqueduct": "workspace:~",
-		"@fluidframework/azure-client-previous": "npm:@fluidframework/azure-client@2.0.0-internal.7.2.0",
+		"@fluidframework/azure-client-previous": "npm:@fluidframework/azure-client@2.0.0-internal.8.0.0",
```

Comment:
I suspect this is going to conflict with a PR by @dannimad to update the typetests. That should probably be its own PR, and we can keep this one focused on what it wants to do instead of bundling a huge change throughout the repo.

Reply:
Huh, actually Daniel's PR was already merged... @kekachmar can you merge main into this branch? Or recreate this change in a new branch from the tip of main? There's a lot of changes showing up in the diff here that already exist in main, and 281 changed files in this one doesn't make sense...

Reply:
Hey @alexvy86, I was working on merging the latest changes, but it seems to have caused more differences. I'll make a new branch with these changes.